Code/Framework/AzFramework/AzFramework/Spawnable/SpawnableEntitiesInterface.h

```diff
@@ -314,6 +320,8 @@ namespace AzFramework
         virtual void ClaimEntities(
             EntitySpawnTicket& ticket, ClaimEntitiesCallback listCallback, ClaimEntitiesOptionalArgs optionalArgs = {}) = 0;
 
+        virtual void ClaimEntity(AZ::EntityId entityId, void* ticket, ClaimEntityOptionalArgs optionalArgs = {}) = 0;
```

Comment:
would be nice to have some comments for these functions in API, I remember using SpawnableEntitiesManager during game jam, and it was a bit confusing to figure out what some functions do/how to properly use it.